.gitignore

```diff
@@ -1 +1,2 @@
 /target
+/.vscode
```

Comment:
I'd rather editor-specific ignores go in a [global git ignore file](https://stackoverflow.com/a/7335487), though I see that might be fighting a [loosing battle](https://github.com/vectordotdev/vector/blob/665ab39dce1cfcad46cbae746c82973df957bfdf/.gitignore#L24).

Reply:
Oh, derp. I can unwind this. I added it out of habit because I needed a custom debug launch configuration and as I was doing Git stuff, I noticed it about to be added and wanted to ignore it.

Long story short.. I'll unwind this. :P